lib/conjur/cli.rb

```diff
@@ -93,6 +93,9 @@ def init!
         apply_config
         load_plugins
         commands_from 'conjur/command'
+      rescue Exception => ex
```

Comment:
Rescuing `Exception` will trap errors such as interrupts and out-of-memory conditions. In Ruby, the `StandardError` class is the class to rescue by default, which you can do with a bare `rescue`.

See - https://robots.thoughtbot.com/rescue-standarderror-not-exception

Reply:
I will change this to `StandardError` instead of `Exception`. I chose not to initially here because I was making the assumption it will still be better to withhold the full stack trace by default for any error.